extension/goext/schemas.go

```diff
@@ -156,6 +156,9 @@ type ISchema interface {
 
 	// RegisterRawType registers a raw resource type, containing db annotations
 	RegisterRawType(rawResourceType interface{})
+
+	// Children returns list of schemas that extend schema with given id
+	Children() []ISchema
```

Comment:
In this sense, children is not proper term even in an object-oriented meaning. Should be `SuperSchemas()` or something